src/main/java/uk/nhs/ctp/service/CarePlanService.java

```diff
@@ -35,4 +37,12 @@
         .collect(Collectors.toList());
   }
 
+  public void completeCarePlans(String[] carePlanIds) {
+    Stream.of(carePlanIds)
+        .map(id -> authenticatedStorageService.get(id, CarePlan.class))
+        .filter(carePlan -> carePlan.getStatus() == CarePlanStatus.DRAFT)
```

Comment:
The status could also be ACTIVE if it's a final care plan. I suspect there's no route at the moment which could trigger this update at the end of triage, but after CDSCT-357 there will be.

Reply:
Good point, I've updated to reflect that we could update if the status is Active as well.